tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -1146,6 +1146,7 @@ public function testRemoveOrphanedManagedElementFromOneToManyExtraLazyCollection
 
         $user->userLists->removeElement($this->_em->find(UserList::class, $userListId));
 
+        $this->_em->flush();
```

Comment:
Yes, it's BC break, but it will become in logic of Doctrine. All changes must be complete with flush.